packages/@aws-cdk/aws-pipes-targets-alpha/README.md

```diff
@@ -181,7 +182,9 @@ The API destination will receive the (enriched/filtered) source payload.
 declare const sourceQueue: sqs.Queue;
 declare const dest: events.ApiDestination;
 
-const apiTarget = new targets.ApiDestinationTarget(dest);
+const apiTarget = new targets.ApiDestinationTarget(dest, {
+  inputTransformation: pipes.InputTransformation.fromObject({ body: "👀" }),
+});
```

Comment:
Ah, when I said to include an `inputTransformation` example, I meant on top of the existing default example 😅 So for instance, for Lambda Function (L127-L173), there's the default example (L131-L141) and the input transformation example (L161-L173). I was hoping we could replicate that structure for the API destination as well.